xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -20,8 +20,45 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>An assembly load context represents the runtime's concept of a scope for binding. This class is abstract.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Runtime.Loader.AssemblyLoadContext> represents a load context. Conceptually, a load context creates a scope for loading, resolving, and potentially unloading a set of assemblies.
+ 
+ <xref:System.Runtime.Loader.AssemblyLoadContext> is an abstract class. The <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)?displayProperty=nameWithType> needs to be implemented to create a concrete class.
+ 
+ The <xref:System.Runtime.Loader.AssemblyLoadContext> exist primarily exists to provide assembly isolation. It allows multiple versions of the same assembly to be loaded within a single process. It replace the isolation mechanisms provided by multiple <xref:System.AppDomain> instances in the .NET Framework.
+
+### Usage in the runtime
+ The runtime implements two <xref:System.Runtime.Loader.AssemblyLoadContext>:
+     * The <xref:System.Runtime.Loader.AssemblyLoadContext.Default?displayProperty=nameWithType> represents the runtimes default binding context which is used for the framework and the application main assembly and its normal dependencies.
+     * The <xref:System.Reflection.Assembly.LoadFile(System.String)?displayProperty=nameWithType> method isolates the assemblies it loads by instantiating the most basic <xref:System.Runtime.Loader.AssemblyLoadContext>.  It has a simplistic isolation scheme which loads each assembly in its own <xref:System.Runtime.Loader.AssemblyLoadContext> with no dependency resolution.
+ 
+ To implement the <xref:System.Reflection.Assembly.LoadFrom(System.String)?displayProperty=nameWithType> event, the runtime adds a <xref:System.Runtime.Loader.AssemblyLoadContext.Resolving> event handler to bind to dependent files in the same directory.
```

Comment:
I assume you meant "LoadFrom method" instead of "LoadFrom event"

Reply:
The runtime registers event handler on `AssemblyLoadContext.AssemblyResolve`. The difference is that `AssemblyResolve` is a global event (fires for every load context), whether the `Resolving` is per load context.

Reply:
Yes I misread this.  It was confusing since `AssemblyLoadContext` does not have an `AssemblyResolve` in the API.  It actually belongs in `AppDomain` in the API.

Reply:
Removing this paragraph as it out of context.